go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.8.1
 	github.com/paypal/gatt v0.0.0-20151011220935-4ae819d591cf
+	github.com/pkg/errors v0.9.1
```

Comment:
we would not use this package, just use Go native error packages instead.

Reply:
Good catch, I've removed that. Maybe I can go make that same fix for `kubeadm` haha